lib/DHCPServerOpMode.pm

```diff
@@ -63,9 +63,16 @@ sub get_active {
             $ip = $1;
         }
         next if (!defined($ip));
-        if ($line =~ /shared-network:\s(.*)/) {
-            $pool = $1;
-        }
+        #if ($line =~ /shared-network:\s(.*)/) {
```

Comment:
Seems like you use tabs for indention instead of spaces (previous version). Could you please fix this? In addition please remove your commented code.

Reply:
Fixed. The intent is weird, I have spaces set in my .vimrc, I could only think it happend when I directly edited the code on the VM running vyos, that wouldn't have a resource file. Sorry for that.